src/app/watch/watch-owner/watch-owner.component.ts

```diff
@@ -7,11 +9,15 @@ import { Component, OnInit, Input } from '@angular/core';
 })
 export class WatchOwnerComponent implements OnInit {
 
-  @Input() owner: string;
+  @Input() owner: User;
+  @Input() channel: Channel;
   @Input() publishDate: string;
 
+  channelLink: string;
+
   constructor() { }
 
   ngOnInit() {
+    this.channelLink = `\\channels\\${this.channel.id}`;
```

Comment:
Use normal slashes in URL instead of back-slash

Reply:
Done